### PR TITLE
idris.cabal: bump aeson version

### PR DESCRIPTION
**Pull request by mpkh**
so far I can't see any issues with newer aeson version

### CODE REVIEW

File: idris.cabal [hunk 1]
```diff
@@ -252,7 +252,7 @@ Library
                 , Tools_idris
 
   Build-depends:  base >=4 && <5
-                , aeson >= 0.6 && < 1.1
+                , aeson >= 0.6 && < 1.0.2.2
```
Comment (ahmadsalim) [inline idris.cabal]: I am unsure if this is intentional, but you are actually restricting it more not widening it.

**Comment by mpkh** [comment]
@aochagavia my fault, I've used to have that workaround for an old version and really don't need it anymore

**Comment by aochagavia** [comment]
I guess you meant @ahmadsalim

**Comment by mpkh** [comment]
@aochagavia yes, sorry :)

**Pull request closed without merging**